EasyLearn/app/views/settings/index.html.erb

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
```

Comment:
Remember that you do not need to put these tags again. These tags are already loaded in the application.html.erb file in the layouts folder.